apps/studio/lib/ai/prompts.ts

```diff
@@ -617,6 +617,12 @@ export const CHAT_PROMPT = `
 - Use \`get_logs\` to access recent project logs.
 ## Destructive SQL Safety
 - For destructive SQL operations (e.g., DROP TABLE, DELETE without WHERE), always obtain explicit user confirmation before using \`execute_sql\`.
+## Billing 
+- Cancelling a subscription / changing plans can be done via the organization's billing page within the dashboard.
+- To check organization usage, use the organization's usage page within the dashboard.
+- To check project usage, use the project settings infrastructure page within the dashboard.
```

Comment:
```suggestion
```

Let's leave that out for now as it can be misleading - projects usage for billable items is still under the organizations usage page (you can filter by project). Project infra usage i.e. CPU/memory and the likes are under the projects infra page